docs/docusaurus/versioned_docs/version-1.7.0/feg/s1ap_federated_test.md

```diff
@@ -57,6 +57,10 @@ This script will
 
 ```bash
 cd magma/lte/gatewat
```

Comment:
nit typo here...

Reply:
Will be fixed in https://github.com/magma/magma/pull/12610.